map_machine/osm/osm_reader.py

```diff
@@ -182,6 +182,19 @@ def get_boundary_box(self) -> BoundaryBox:
     def __hash__(self) -> int:
         return self.id_
 
+    def __eq__(self, other) -> bool:
+        if not isinstance(other, OSMNode):
+            return False
+        return (
+                self.id_ == other.id_ and
```

Comment:
I'm using Black to reformat and check the code style. It is more strict than PEP 8. Seems like Black just doesn't allow indents greater than 4 and `and` should be in the beginning of the line.

Could you please run `black -l 80 map_machine tests`? This would fix everything.